### ISSUE TITLE
Please add MAKEFILE to build + instruction/docker setup

### ISSUE THREAD

**Issue by daic11** [SafePalWallet/safepal-x1#2]
The build is impossible due to the lack of the MAKEFILE.
There are also no instruction about the environment variables/docker container required to make build possible.
Please update the repo with the building instruction. It looks like a random folder from the desktop:)

Hope you will update the repo thank you!

P.S.: there is a lack of coin management in your last 1.0.5 firmware version. You can't remove addresses/coins that you added through the Safepal app. So please modify. There are also bluetooth connection problems with IOS (it connects randomly).

Thank you.

**Comment by eliasfelipedasilva** [comment]
We need more instructions about the enviroment, please, provide to us.

**Comment by crypto88love** [comment]
Yes 1.03,4,5,6 X1 firmware updates not available on GitHub, No Reply from Dev
